awxkit/awxkit/api/pages/schedules.py

```diff
@@ -47,6 +47,12 @@ def remove_credential(self, cred):
         with suppress(exc.NoContent):
             self.related.credentials.post(dict(id=cred.id, disassociate=True))
 
+    def add_label(self, label):
+        if isinstance(label, page.Page):
```

Comment:
@AlanCoding what is the better pattern here? Use label object or id? I see for example add_credential method above expects the id.